TabloidCLI/Repositories/BlogRepository.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Data.SqlClient;
+using TabloidCLI.Models;
+using TabloidCLI.Repositories;
+
+namespace TabloidCLI
+{
+    public class BlogRepository :DatabaseConnector, IRepository<Blog>
+    {
+        public BlogRepository(string connectionString) : base (connectionString) { }
+
+        public List<Blog> GetAll()
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"SELECT Id, Title, URL
+                                        FROM Blog";
+
+                    List<Blog> blogs = new List<Blog>();
+
+                    SqlDataReader reader = cmd.ExecuteReader();
+                    while (reader.Read())
+                    {
+                        Blog blog = new Blog()
+                        {
+                            Id = reader.GetInt32(reader.GetOrdinal("Id")),
+                            Title = reader.GetString(reader.GetOrdinal("Title")),
+                            Url = reader.GetString(reader.GetOrdinal("URL")),
+                        };
+                        blogs.Add(blog);
+                    }
+
+                    return blogs;
+                }
+            }
+        }
+
+        public Blog Get(int id)
+        {
+            Blog blog = null;
+            return blog;
+        }
+
+        public void Insert(Blog entry)
+        {
+
```

Comment:
personal preference: as i work forward and have unused methods i like to make sure and leave in `throw new NotImplementedException();` so that the compiler can build the program and it can be tested.... did this get tested before this pr was opened?

Reply:
Yessir, tested it and it ran with no errors.  I just needed to meet the "contract obligations" of the interface and didn't even think about throwing the exception.  Will use that going forward.

Reply:
word... as long as it runs the code looks good. :shipit: